### DISCUSSION TITLE
'g++' could not be spawned. Is it installed and on your path? If so please open an issue on the package spawning the process.

### DISCUSSION THREAD

**Discussion by Rasel-BD** [atom/atom#23393] [Q&A]
I installed atom,minGW and cpp compiler and also edited system variables,but i can not run program,please help [me,
![image](https://user-images.githubusercontent.com/77558559/146881482-6a1ec99f-4b8e-4c52-a3a4-2be05f860441.png)

**Comment by icecream17** [comment]
You added it to the minGW variable, but did you add it to the Path variable? (4th row)
